application/src/Controller/DefaultController.php

```diff
@@ -8,11 +8,17 @@
 
 class DefaultController extends Controller
 {
-    public function indexAction(Weekend $weekend)
+    public function index(Weekend $weekend)
     {
+        if(!preg_match('/AppleWebKit|Chrome|Edge|Gecko|Opera|Trident/i', $_SERVER['HTTP_USER_AGENT'])) {
```

Comment:
```suggestion
    public function index(Weekend $weekend, Request $request)
    {
        if(!preg_match('/AppleWebKit|Chrome|Edge|Gecko|Opera|Trident/i', $request->headers->get('User-Agent'))) {
```
And you'll have to add the `use ...\Request`

I did not try the code, but it should be something similar

Reply:
Done, thanks! 👌